site/en/docs/extensions/mv3/xhr/index.md

```diff
@@ -1,18 +1,13 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Cross-origin XMLHttpRequest"
-seoTitle: "Chrome Extensions: Cross-origin XMLHttpRequest"
+title: "Cross-origin Network Requests"
+seoTitle: "Chrome Extensions: Cross-origin Network Requests"
 date: 2012-09-18
-updated: 2020-03-09
-description: How to implement cross-origin XHR in your Chrome Extension.
+updated: 2023-05-23
+description: How to implement cross-origin network requests in your Chrome Extension.
 ---
 
-{% Aside 'warning' %}
-In Manifest V3, `XMLHttpRequest` is not supported in background pages (provided by Service Workers).
-Please consider using its modern replacement, `fetch()`.
-{% endAside %}
-
-Regular web pages can use the [`XMLHttpRequest`][1] object to send and receive data from remote
```

Comment:
In lieu of this, I think you need a new heading farther down.

- Give it a name like "What about XMLHttpRequest()?".
- Explain that XHR() is supported for backwards compatibility.
- New work should favor fetch().
- Extensions service workers don't support XHR, but calling it in an extension triggers the SW's fetch handler.